src/app/services/graph-constants.ts

```diff
@@ -11,6 +11,8 @@ export enum PERMS_SCOPE {
     APPLICATION = "Application",
     PERSONAL = "DelegatedPersonal"
 };
-export const DISPLAY_DELEGATED_PERMISSIONS = true;
-export const DISPLAY_APPLICATION_PERMISSIONS = false;
+export enum PERMISSION_MODE_TYPE {
+    TeamsApp,
+    User
+}
```

Comment:
ae6927dc2910956fd1323e57a64fa1e1a5c6830a Re-ordered `User, TeamsApp` to `TeamsApp, User` to fix wrong mode bug discovered by Lucy (i.e. asking to switch to User mode when you're already in User mode).

Reply:
I was actually looking for this bug too! That's a huge find, thank you Jerry 🙏